pages/controls/Subresource Integrity

```diff
@@ -0,0 +1,105 @@
+---
+
+layout: col-sidebar
+title: Subresource Integrity (SRI)  
+author: Vaibhav Malik
+tags: [controls, application, web]
+permalink: /controls/SubresourceIntegrity
+
+---
+
+{% include writers.html %}
+
+
+## Description
+
+Subresource Integrity (SRI) is a security feature that enables browsers to verify that 
+the resources they fetch (for example, from a CDN) are delivered without unexpected 
+manipulation. It allows you to provide a cryptographic hash that a fetched resource must match.
```

Comment:
This seems directly ripped from: https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity

Reply:
Sure. Adding in references. Also changed